src/utils.js

```diff
@@ -589,6 +589,44 @@ const getFeaturetypes = (dataset) => {
   return false;
 };
 
+/**
+ * Checks if a response is one of the supported binary file types (video and image)
+ *
+ * @param {object} response response from Seldon
+ * @returns {boolean} if a response includes a encoded base64 string or not
+ */
+const isSupportedBinaryData = (response) => {
+  const data = Object.values(response).shift();
+
+  try {
+    const [base, content] = data.split(',');
+    const pattern = /[A-Za-z0-9+/=]/;
+
+    const mimeType = base
+      .match(/[^:]\w+\/[\w-+\d.]+(?=;|,)/)[0]
+      .split('/')
+      .shift();
+
+    if (['video', 'image'].includes(mimeType) && pattern.test(content))
+      return true;
+  } catch (error) {
+    return false;
+  }
+};
```

Comment:
Este `try` `catch` é só por causa do `.match(/[^:]\w+\/[\w-+\d.]+(?=;|,)/)[0]` ?
Se for isso, faça um `if != null`  com o retorno do match.